app/src/main/java/com/example/weather/network/Dao.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.weather.network
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+interface CityDao {
+
+    @Query("SELECT * FROM city_table ORDER BY name ASC")
+    fun getCities(): List<CachedCityOverview>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    suspend fun insert(cachedCity: CachedCityOverview)
+
+    @Query("DELETE FROM city_table")
+    suspend fun deleteAll()
+}
+
+@Dao
+interface CityDataDao {
+
+    @Query("SELECT * FROM city_table ORDER BY name ASC")
```

Comment:
Do we want to fetch all the data from the table? Or just the one city we are looking for details data for